packages/amplify-category-notifications/src/commands/notifications.ts

```diff
@@ -0,0 +1,51 @@
+import { $TSAny, $TSContext } from 'amplify-cli-core';
+import { printer } from 'amplify-prompts';
+
+export const name = 'notifications';
+export const alias = ['notification'];
+
+/**
+ * Displays Notifications walkthrough help for sub-commands ( routes to sub-commands on test/windows platform)
+ * @param context amplify cli context
+ * @returns Returns Notifications feature flow output in test/windows environment , and undefined otherwise
+ */
+export const run = async (context: $TSContext) : Promise<$TSAny|undefined> => {
+  if (/^win/.test(process.platform)) {
+    try {
+      // eslint-disable-next-line import/no-dynamic-require, global-require, @typescript-eslint/no-var-requires
+      const notificationsFlow = require(`./${name}/${context.parameters.first}`);
```

Comment:
You may be able to eliminate the eslint warnings using Typescripts dynamic import:

`await import(`./${name}/${context.parameters.first}`);`

Reply:
https://mariusschulz.com/blog/dynamic-import-expressions-in-typescript#using-the-await-operator-with-import

Reply:
nice